contents/handbook/engineering/incidents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Handling an incident
+sidebar: Handbook
+showTitle: true
+---
+
+Incidents are going to happen.
+
+## When to raise an incident
+
+**When in doubt, raise an incident.** We'd much rather have declared an incident which turned out not to be an incident. Many incidents take too long to get called, or are missed completely because someone didn't ring the alarm when they had a suspicion something was wrong.
+
+To declare an incident, type `/incident` in the #incidents channel. This will create a new channel and send updates.
+
+Anyone can declare an incident, including non-engineers. If in doubt, check with your nearest engineer.
+
+Some things that should definitely be an incident
+- `app.posthog.com` being completely unavailable (not just for you)
+- No insights can be created
+- Feature flags are not being returned at all, or `/decide` is down
+- Various alerts defined as critical, such as disk space full, OOM or >5 minute ingestion lag
+
+Things that _shouldn’t_ be an incident
+- Insights returning incorrect data
+- Events being < 5 minutes behind
+- Unable to save insights, create feature flags
+
+## What happens during an incident
+
+The person who raised the incident is the incident lead. It’s their responsibility to:
+- Make sure the right people join the zoom. This includes the current on call person (https://posthog.pagerduty.com/service-directory/P43Y0E8), people from Infra and the feature owner if relevant (https://posthog.com/handbook/engineering/feature-ownership)
```

Comment:
Not sure how `incident.io` works but we should have a permanent Google Meet/Zoom room that people can bookmark and go there in case of issues.

Reply:
Incident.io does this automatically
<img width="562" alt="image" src="https://user-images.githubusercontent.com/1727427/186622075-3ea2f9ba-94a1-4840-87f2-202ed564ecb0.png">